samples/net/eth_native_posix/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _eth-native-posix-sample:
+
+Native Posix Ethernet
+#####################
+
+Overview
+********
+
+The eth_native_posix sample application for Zephyr creates **zeth** network
+interface to host system. One can communicate with Zephyr via this network
+interface.
+
+The source code for this sample application can be found at:
+:file:`samples/net/eth_native_posix`.
+
+Building And Running
+********************
+
+To build the eth_native_posix sample application, follow the steps
+below.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/eth_native_posix
+   :board: native_posix
+   :conf: <config file to use>
+   :goals: build
+   :compact:
+
+Normally one needs extra priviliges to create and configure the TAP device in
+host system. So you would need to execute the Zephyr process with admin
+priviliges for example like this:
+
+.. code-block:: console
+
+    $ sudo --preserve-env=ZEPHYR_BASE make -s -C outdir/native_posix run
```

Comment:
Maybe a silly question: Would it be possible to run 2 instances of this application back to back?

Reply:
Absolutely, one could have multiple config files with properly prepared IP addresses in them. I will add command line options passing to default setup file (net_host_setup) that will help to meet this goal.